### DISCUSSION TITLE
.

### DISCUSSION THREAD

**Discussion by ghost** [mastodon/mastodon#21513] [General]
I've just setup a private mastodon server, and the admin user and any user I create is not visible from other servers.
Whenever i try to follow someone on a different instance i get a "Withdraw follow request" instead of following them and seeing their posts on my timeline.

after following someone, i've noticed that i keep getting a 401 error in sidekeq, eg, 
```
Mastodon::UnexpectedResponseError: https://mastodon.online/users//inbox returned code 401
Stoplight::Error::RedLight: https://mastodon.online/users//inbox
```

**Comment by n33d4n4m3** [comment]
This might be helpful: https://github.com/mastodon/mastodon/issues/7086

**Comment by ghost** [comment]
weird, i've created another user on this instance and it's working fine and can be seen on other servers, but the original handle i used the first time still can't be seen, weird.
